src/test/resources/org/broadinstitute/barclay/help/templates/BashTestDoclet/bash-completion.template.ftl

```diff
@@ -0,0 +1,343 @@
+
+####################
+# Tab completion file to allow for easy use of this tool with the command-line (bash/zsh)
+####################
+
+<#include "bash-completion.macros.ftl"/>
+
+####################################################################################################
+
+# High-level caller/dispatch script information:
```

Comment:
I just realized that this is a test resource, which is fine as it is...but I would like to have a default template for bash-completion which is simple enough to use for any Barclay-derived toolkit...